storybook/.storybook-react/config.js

```diff
@@ -1,8 +1,8 @@
-import '!style-loader!css-loader!sass-loader!import-glob-loader!@nypl/design-system-styles/style.scss';
-import { configure, addDecorator } from '@storybook/react';
-import { withInfo } from '@storybook/addon-info';
-import { withKnobs } from '@storybook/addon-knobs';
-import { withA11y } from '@storybook/addon-a11y';
+import "!style-loader!css-loader!sass-loader!import-glob-loader!@nypl/design-system-react-components/style.scss";
```

Comment:
import-glob-loader is not needed here anymore, right?

Reply:
I forgot about base styles - I think globbing for base styles is fine then, since it all lives in the same place.

Reply:
I forgot too until you mentioned the dependency, hah